src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -29,6 +29,13 @@ clang_c_languaget::clang_c_languaget()
 {
   // Build the compile arguments
   build_compiler_args(clang_headers_path());
+
+#if 0
```

Comment:
Do you know if we need to keep this comment code here?

Reply:
The code is useful for debugging the clang invocation params. I've put it behind Verbosity::Debug.